cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -108,6 +108,8 @@ elseif(OPENWRT)
   set(configure_opts "linux-generic32" "no-async")
 endif()
 
+string(REGEX REPLACE "^ " "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
```

Comment:
Please update `PACKAGE_INTERNAL_DEPS_ID` for OpenSSL so we can test this change. See:
- https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html